### PR TITLE
updated to remove newline delimiter from json

### PR DESCRIPTION
**Pull request by jdnurme**
Alter json formatting to try and repair kettle

### CODE REVIEW

**Comment by MushuEE** [comment]
Will test in staging and update results here. Staging is not as aggressive as Prod and might not hit the problem tests, but it should give us an idea if this is a breaking change or not. 

/assign

File: kettle/update.py [hunk 1]
```diff
@@ -62,21 +62,21 @@ def main():
         mj_ext = ' --reset-emitted'
 
     if os.getenv('DEPLOYMENT', 'staging') == "prod":
-        call(f'{mj_cmd} {mj_ext} --days {DAY} | pv | gzip > build_day.json.gz')
+      call(f'{mj_cmd} {mj_ext} --days {DAY} | jq -c '.[]' | pv | gzip > build_day.json.gz')
```
Comment (MushuEE) [inline kettle/update.py:65]: The '' need to be escaped 

`jq -c '\.[]\'`

Comment (jdnurme) [reply kettle/update.py]: Done

**Comment by MushuEE** [comment]
#23135

**Comment by spiffxp** [comment]
/lgtm cancel
Please pass tests
```
Linting python...
************* Module kettle.update
kettle/update.py:66:8: E0001: unexpected indent (<unknown>, line 66) (syntax-error)
```

**Pull request closed without merging**